config/config.go

```diff
@@ -14,9 +14,12 @@ type Config struct {
 	GracefulShutdownTimeout    time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
 	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
+	DefaultRequestTimeout      time.Duration `envconfig:"DEFAULT_REQUEST_TIMEOUT"`
```

Comment:
Need to add these three new fields to [the config test too](https://github.com/ONSdigital/dp-cantabular-metadata-exporter/blob/731da9e648fd87be63c4e2f981d9b53ba6ea9176/config/config_test.go).

Reply:
Also the README for the service is missing most of the config variables. Please add them all in.